packages/expo-dev-launcher/babel.config.js

```diff
@@ -37,12 +41,19 @@ module.exports = function (api) {
     alias['react-native-safe-area-context'] = safeAreaContext;
   }
 
+  if (reanimated) {
```

Comment:
should we throw an error here if this is null/falsy?

Reply:
Actually, it's a wanted behaviour. Right now, it's easy to swap reanimated version and use the one from the repo. It makes the updating process easier because you can always test something without running vendoring script.

Reply:
Also, `tryResolveModule` function will print an error message ;)